datacommons/datacommons.py

```diff
@@ -397,6 +398,28 @@ def get_observations(self,
         'Observation',
         max_rows=max_rows)
 
+  # -------------------------- CACHING FUNCTIONS --------------------------
+
+  def save_dataframe(self, pd_dataframe, file_name):
+    """Saves pandas dataframe for later retrieving.
```

Comment:
Since this is user-facing. How about adding some comments to say this is saved within a user's namespace and is not currently accessible to any other user?